dist/dss.js

```diff
@@ -1,53 +1,56 @@
 //File : src/dss.js
 
-
-(function(world){
+(function(world) {
   'use strict';
 
   /**
    * [dss description]
    * @return {[Object]}
    */
-  world.dss = function(){ 
-
+  world.dss = function() {
     return {
       core : {}
     };
-
   }();
-
 })(this);
+
 //File : src/core/dss.core.ajax.js
 
-(function(dss){
+(function(dss) {
 	'use strict';
 
- dss.core.ajax = function(args){
- 	var callback = args.callback || function(){return false;};
+ dss.core.ajax = function(args) {
+ 	var callback = args.callback || function() { return false; };
  	var url = args.url;
  	var method = args.method || 'GET';
  	var data = args.data || false;
-
  	var xhr;
- 	
- 	if(typeof XMLHttpRequest !== 'undefined') xhr = new XMLHttpRequest();
- 	else {
- 		var versions = ["MSXML2.XmlHttp.5.0", 
- 		"MSXML2.XmlHttp.4.0",
- 		"MSXML2.XmlHttp.3.0", 
- 		"MSXML2.XmlHttp.2.0",
- 		"Microsoft.XmlHttp"];
- 		
+
+ 	if (typeof XMLHttpRequest !== 'undefined') {
+    xhr = new XMLHttpRequest();
+  } else {
+ 		var versions = [
+      'MSXML2.XmlHttp.5.0',
```

Comment:
Good observation, the original file have that problem: https://github.com/guisouza/dss/blob/master/dist/dss.js

Send a PR.